src/ophys_etl/modules/segmentation/utils/roi_utils.py

```diff
@@ -321,3 +323,57 @@ def roi_list_from_file(file_path: pathlib.Path) -> List[OphysROI]:
             roi = OphysROI.from_schema_dict(roi_data)
             output_list.append(roi)
     return output_list
+
+
+def select_contiguous_region(
```

Comment:
please consider that this might be simpler to digest than the implementation here:
```
import numpy as np
from skimage.measure import label


def contiguous(im, point):
    labels = label(im)
    ilabel = labels[point[0], point[1]]
    connected = np.zeros_like(im)
    connected[labels == ilabel] = 1
    return connected

x = np.zeros((10, 10))
x[0:2, 0:2] = 1
x[5:7, 5:7] = 1

print(x)
print(contiguous(x, [1, 1]))
print(contiguous(x,  [6, 6]))
```

Reply:
I have changed the implementation